.github/workflows/_61_upload_engine_shared_libs.yml

```diff
@@ -0,0 +1,51 @@
+on:
+  workflow_call:
+
+permissions:
+  id-token: write
+  contents: read
+jobs:
+  upload:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout 🛒
```

Comment:
why do we need to checkout here? we aren't doing anything with the repo as far as I can tell?